autoreduce_frontend/templates/snippets/variables/render.html

```diff
@@ -3,36 +3,36 @@
 
 {% for name, variable in variables.items %}
 <div class="form-group">
-    <label for="var-{{ type }}-{% encode_b64 name %}" class="control-label col-md-3">
+    <label for="var-{{ type }}-{% encode_b64 name %}" class="col-form-label col-md-3">
```

Comment:
```suggestion
    <label for="var-{{ type }}-{% encode_b64 name %}" class="col-form-label col-12">
```
Makes longer variable names fit better, now that they are on top and not on the left!